### PR TITLE
Add native version of _.each for objects

### PR DESCRIPTION
**Pull request by chipironcin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -858,6 +859,32 @@ Iterates over a list of elements, yielding each in turn to an iteratee function.
 
 **[⬆ back to top](#quick-links)**
 
+### _.each(object)
```
Comment (stevemao) [inline README.md]: I'd change the title to _.each (for object) to avoid confusion.

Comment (stevemao) [reply README.md]: The title is not changed yet.

File: README.md [hunk 2]
```diff
@@ -858,6 +859,32 @@ Iterates over a list of elements, yielding each in turn to an iteratee function.
...
+
+  ```js
+  // Underscore/Lodash
+  _.each({ 'a': 1, 'b': 2, 'c': 3 }, function(value, key) {
+    console.log(key);
+  });
+  // output: a b c
+
+  // Native
+  Object.entries({ 'a': 1, 'b': 2, 'c': 3 }).forEach(function ([key, value]) {
```
Comment (ODudek) [inline README.md]: Hmmm... I wonder, wouldn't be better to use forin loop? It has support for more browsers than `Object.entries`

Comment (pldilley) [reply README.md]: Well, perhaps the same argument could be made for arrays too (for-loop vs .forEach)...

Could the following be a compromise?
```
var obj = { 'a': 1, 'b': 2, 'c': 3 };
Object.keys(obj).forEach(function(key) {
  var value = obj[key];
  console.log(key, value);
});
```
^ IE9+ supported them, in line with the current suggested array solution.

**Comment by cht8687** [comment]
@chipironcin many thanks to this PR. 
could you address the two PR reviews and commit changes? so that we can merge this. thanks.

**Comment by GeorgeWL** [comment]
@chipironcin looks to be all good, think you could fix the conflicts?

**Comment by chipironcin** [comment]
Yes, I just need to find some time to fix this properly. Given we are in #Hacktoberfest I will fix this very soon. Sorry for the delay.

**Comment by cjtaylor1990** [comment]
I actually added this in #231 without noticing this pull request (sorry!). I realize though that I will likely re-write the README to include _.each in both the Object section and the Array section, given that the sections are broken up by data type.

**Comment by stevemao** [comment]
Yeah. Thanks!

**Pull request closed without merging**